code/datums/components/footstep.dm

```diff
@@ -27,13 +31,75 @@
 			v /= 2
 			e -= 5
 	steps++
-	if(steps >= 3)
+	
+	if(steps >= 6)
```

Comment:
This is an undocumented revert of citadel-specific changes. This will result in footsteps playing once every two steps instead of once every three steps, which sounds absolutely horrendous while sprinting.

Reply:
I'll fix that, but tbh I liked the old tg-style footstep frequency.